modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -361,6 +361,52 @@ public void testSendNotification() throws Exception {
 				user2.getEmailAddress()));
 	}
 
+	@Test
+	public void testSendNotificationWithRegularRoles() throws Exception {
+		Role role1 = _addRole(RoleConstants.TYPE_REGULAR, user1);
+		Role role2 = _addRole(RoleConstants.TYPE_REGULAR, user2);
+
+		NotificationTemplate notificationTemplate =
+			notificationTemplateLocalService.addNotificationTemplate(
+				NotificationTemplateUtil.createNotificationContext(
+					TestPropsValues.getUser(), 0, RandomTestUtil.randomString(),
+					RandomTestUtil.randomString(),
+					NotificationTemplateConstants.EDITOR_TYPE_RICH_TEXT,
+					Arrays.asList(
+						createNotificationRecipientSetting(
+							NotificationRecipientSettingConstants.NAME_CC,
+							"[%CURRENT_USER_EMAIL_ADDRESS%],cc@liferay.com"),
+						createNotificationRecipientSetting(
+							NotificationRecipientSettingConstants.NAME_FROM,
+							"[%CURRENT_USER_EMAIL_ADDRESS%]"),
+						createNotificationRecipientSetting(
+							NotificationRecipientSettingConstants.
+								NAME_FROM_NAME,
+							Collections.singletonMap(
+								LocaleUtil.US, "[%CURRENT_USER_FIRST_NAME%]")),
+						createNotificationRecipientSetting(
+							NotificationRecipientSettingConstants.
+								NAME_SINGLE_RECIPIENT,
+							Boolean.FALSE.toString()),
+						createNotificationRecipientSetting(
+							NotificationRecipientSettingConstants.NAME_TO,
+							role1.getName()),
+						createNotificationRecipientSetting(
+							NotificationRecipientSettingConstants.NAME_TO_TYPE,
+							NotificationRecipientConstants.TYPE_ROLE)),
+					RandomTestUtil.randomString(),
+					NotificationConstants.TYPE_EMAIL, Collections.emptyList()));
+
+		_testSendNotificationWithRegularRole(
+			StringPool.BLANK, 0, user1.getEmailAddress(), notificationTemplate);
+
+		_roleLocalService.addUserRole(user1.getUserId(), role1.getRoleId());
+		_roleLocalService.addUserRole(user2.getUserId(), role2.getRoleId());
+
+		_testSendNotificationWithRegularRole(
+			StringPool.BLANK, 1, user1.getEmailAddress(), notificationTemplate);
```

Comment:
Hey @rijema I believe we can use the method `_testSendNotificationWithRoles` by passing `null` as an argument of the parameter `accountEntry`, could you please try to do this, if it works delete the method `_testSendNotificationWithRegularRole`.

Thank you!